interactions/api/models/channel.py

```diff
@@ -349,10 +352,14 @@ async def modify(
             self.rate_limit_per_user if rate_limit_per_user is MISSING else rate_limit_per_user
         )
         _position = self.position if position is MISSING else position
-        _parent_id = int(self.parent_id) if parent_id is MISSING else int(parent_id)
+        _parent_id = (
+            (int(self.parent_id) if self.parent_id else None)
+            if parent_id is MISSING
+            else int(parent_id)
+        )
         _nsfw = self.nsfw if nsfw is MISSING else nsfw
         _permission_overwrites = (
-            self.permission_overwrites
+            [_._json for _ in self.permission_overwrites]
```

Comment:
Shouldn't `_` only be used if you are not going to use it?
```suggestion
            [overwrite._json for overwrite in self.permission_overwrites]
```